### DISCUSSION TITLE
Add an option to configure sameSite cookie argument

### DISCUSSION THREAD

**Discussion by dvakatsiienko** [keystonejs/keystone#5330] [Feature requests]
Currently the [`sameSite` cookie argument is hardcoded with a `lax` value](https://github.com/keystonejs/keystone/blob/master/packages-next/keystone/src/session/index.ts#L145).

My frontend is deployed to Vercel and keystone-next backend to Digital Ocean. 
Since authentication involves cookies it makes impossible to have backend and frontend separated because in this case cookie gets blocked by cross-domain cookie policy...

![image](https://user-images.githubusercontent.com/15727958/107681964-6c2b1380-6ca8-11eb-8fdf-0c71e7a41d68.png)

Is it somehow possible to avoid?
Or my feature suggest is to add possibility to configure `sameSite` arg for `statelessSessions` function.

**Comment by gautamsi** [comment]
this is already in place, you have more control in session sameSite option - https://keystonejs.com/docs/apis/session#stateless-vs-stored-sessions
